scraper/ordoclic.py

```diff
@@ -113,8 +111,7 @@ def cp_to_insee(cp):
     # on charge la table de correspondance cp/insee, une seule fois
     global insee
     if insee == {}:
-        with open("data/input/codepostal_to_insee.json") as json_file:
-            insee = json.load(json_file)
+        insee = json.loads(Path("data/input/codepostal_to_insee.json").read_text())
```

Comment:
Perso je préfère quelques lignes supplémentaires pour être plus claire (simple préférence personnelle)

```suggestion
        insee_json_path = Path("data/input/codepostal_to_insee.json")
        with insee_json_path.open() as json_file:
            insee = json.load(json_file)
```

Reply:
Ici, je suis d'accord que d'avoir la création du `Path()` au milieu du `json.loads()` peut être un peu lourd et que ça voudrait la peine de le séparer en deux. Mais ça reste du nitpicking.